proxies/live/apiproxy/policies/AssignMessage.SetAuthorizationRedirectNHSLogin.xml

```diff
@@ -2,7 +2,7 @@
   <Set>
     <StatusCode>302</StatusCode>
     <Headers>
-      <Header name="Location">{identity-service-config.nhs_login.authorize_endpoint}?response_type={request.queryparam.response_type}&amp;client_id={identity-service-config.nhs_login.client_id}&amp;redirect_uri={identity-service-config.nhs_login.redirect_uri}&amp;scope=openid profile&amp;state={apigee.state}&amp;max_age=300</Header>
+      <Header name="Location">{identity-service-config.cis2.authorize_endpoint}?response_type={request.queryparam.response_type}&amp;client_id={identity-service-config.cis2.client_id}&amp;redirect_uri={identity-service-config.cis2.redirect_uri}&amp;scope=openid%20nationalrbacaccess&amp;state={apigee.state}&amp;max_age=300</Header>
```

Comment:
this is called SetAuthorizationRedirectNHSLogin but is redirecting you to cis2 ?